docs/reference/workflows/ecs/interaction/authority.md

```diff
@@ -2,114 +2,112 @@
 
 # ECS: Authority
 
- _This document relates to the [ECS workflow]({{urlRoot}}/reference/workflows/overview.md)._
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
 
-**Authority** is how SpatialOS represents which worker instances can write to each specific [SpatialOS component]({{urlRoot}}/reference/glossary#spatialos-component).
+* [Authority in SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/interest-authority#authority-also-known-as-write-access)
+* [Understanding read and write access](https://docs.improbable.io/reference/latest/shared/design/understanding-access)
+")%>
 
-> If you don't know about authority, you should read [Understanding access](https://docs.improbable.io/reference/latest/shared/design/understanding-access) in the SpatialOS documentation.
->
-> Crucially: at runtime, your worker instance may or may not have authority over a given SpatialOS component.
+Authority is how SpatialOS represents which worker instances can write to a specific [SpatialOS component]({{urlRoot}}/reference/glossary#spatialos-component).
 
-## How authority is represented in Unity's ECS
+## How authority is represented
 
-### Authority shared components
-
-For every [SpatialOS component]({{urlRoot}}/reference/glossary#spatialos-component) attached to a SpatialOS entity, the GDK attaches an _ECS shared component_ (an ECS `ISharedComponentData`) to the ECS entity. This component is called `ComponentName.ComponentAuthority`, where `ComponentName` is the name of the SpatialOS component.
+For every [SpatialOS component]({{urlRoot}}/reference/glossary#spatialos-component) on a checked out SpatialOS entity, the GDK attaches a component to the ECS entity. This component is called `<ComponentName>.ComponentAuthority`, where `<ComponentName>` is the name of the SpatialOS component.
```

Comment:
I know I suggested `<ComponentName>`, but on other pages I realized that we are using `{component name}`. Probably should decide which one we wanna use. I don't have a strong preference as long as it's consistent throughout the pages you touched in this PR 

Edit: just saw the remark you made somewhere else. why would the flow not work with this?

Reply:
changed this bit to ```, the GDK attaches a component to the ECS entity of the type `{component name}.ComponentAuthority`.```